lib/google_scraping/account/helpers/user_token.ex

```diff
@@ -0,0 +1,39 @@
+defmodule GoogleScraping.Accounts.UserToken do
+  use Ecto.Schema
+  alias GoogleScraping.Accounts.UserToken
+
+  @rand_size 32
+
+  schema "users_tokens" do
+    field :token, :binary
+    field :context, :string
+    field :sent_to, :string
+    belongs_to :user, GoogleScraping.Accounts.User
```

Comment:
Lets add a new line for associated columns for better redability
```suggestion

    belongs_to :user, GoogleScraping.Accounts.User
```